src/pipgrip/pipper.py

```diff
@@ -268,12 +276,18 @@ def _download_wheel(package, index_url, extra_index_url, pre, cache_dir):
                     fname = fnames[0]
             else:
                 # wheel was fetched
-                fname = (
-                    line.split(
-                        abs_cache_dir if abs_cache_dir in line else cache_dir, 1
+                # match on lowercase line for windows compatibility
+                fname_len = len(
+                    line_lower.split(
+                        abs_cache_dir_lower
+                        if abs_cache_dir_lower in line_lower
+                        else cache_dir_lower,
+                        1,
                     )[1].split(".whl", 1)[0]
                     + ".whl"
                 )
+                fname = line[-fname_len:]
```

Comment:
@sschuberth could you give this branch a go?
```sh
pip uninstall pipgrip && pip install git+https://github.com/ddelange/pipgrip.git@extended-windows-support
```

Reply:
Thanks, that basically seems to work (it just took a really long time to complete), and I'm getting weird chars instead of an ASCII tree:

```
$ pipgrip --tree pipgrip
pipgrip (0.4.1)
Ôö£ÔöÇÔöÇ anytree (2.8.0)
Ôöé   ÔööÔöÇÔöÇ six>=1.9.0 (1.15.0)
Ôö£ÔöÇÔöÇ click (7.1.2)
Ôö£ÔöÇÔöÇ enum34 (1.1.10)
Ôö£ÔöÇÔöÇ packaging>=17 (20.4)
Ôöé   Ôö£ÔöÇÔöÇ pyparsing>=2.0.2 (2.4.7)
Ôöé   ÔööÔöÇÔöÇ six (1.15.0)
Ôö£ÔöÇÔöÇ pip>=7.1.0 (20.1.1)
Ôö£ÔöÇÔöÇ pkginfo>=1.4.2 (1.5.0.1)
Ôö£ÔöÇÔöÇ setuptools>=38.3 (44.1.1)
Ôö£ÔöÇÔöÇ typing (3.7.4.3)
ÔööÔöÇÔöÇ wheel (0.34.2)
```

Which reminds me of another request of mine: Could you make `--tree` work together with `--json`, so other programs could process the dependency tree easily?

Reply:
ref #31